### DISCUSSION TITLE
fuzzy screen when using sanboot in ipxe

### DISCUSSION THREAD

**Discussion by smtadmin67** [ipxe/ipxe#1003] [General]
hi
i boot Sergei Strelec 2023.06.25 with ipxe, when the device boots up, the monitor according i attached images it's fuzzy, i tried with winpe10 and winpe11 in different devices and also changed resolution in 800*600 before but it's keep going on.
doyou have any ideas?!
thanks
![IMG_20230722_121445](https://github.com/ipxe/ipxe/assets/120118099/a1899928-fbab-4934-893c-421625cc99a1)
![IMG_20230722_121737](https://github.com/ipxe/ipxe/assets/120118099/7afc2afe-08d1-40d2-9398-fb967d9d5d4c)

**Comment by NiKiZe** [comment]
Duplicate of #1002
Opening duplicate issues makes it even less likely to get good help.
